deployer/scripts/install.sh

```diff
@@ -312,7 +312,7 @@ function generate_es_template(){
     --param ES_RECOVER_EXPECTED_NODES=${es_recover_expected_nodes} \
     --param ES_RECOVER_AFTER_TIME=${es_recover_after_time} \
     --param STORAGE_GROUP_DEFAULT=${storage_group} \
-    $image_params
+    "$image_params"
```

Comment:
Probably not.  Any of the deployer related code is in support of 'oc cluster up' which in theory would not be performing upgrade IMO.  Not sure if you can even tell 'oc cluster up' to 'upgrade'